guake/terminal.py

```diff
@@ -478,15 +478,23 @@ def handleTerminalMatch(self, matched_string):
         if value:
             return value
 
+    def get_link_under_terminal_cursor(self) -> Optional[str]:
```

Comment:
Type hints would be a nice thing to do but probably should be consistent and everything else isn't hinted at the current moment, should probably be consistent with the rest of the file. Moving files onto hints could be a good bigger project though.

Reply:
Actually I wouldn't mind starting to add some more type hints to the codebase. Surely not everything, but a good chunk of the low hanging fruits just to get things started. If that's something you'd be interested in.

Reply:
Yeah that'd be nice, can take that if you're willing to do it. Just need the release note file generated by `make reno SLUG=[descriptive_label]` filled in and I'll merge